src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const longTermDiskont = 50;
+  const shortTermDiskont = 20;
```

Comment:
Similarly, 'shortTermDiskont' has a typo and should be corrected to 'shortTermDiscount'. Consistent and correct naming is important for code readability and maintainability.